app/utils/sanityLoader.ts

```diff
@@ -15,7 +15,7 @@ const sanityKlient = createClient({
 
 export const hentDataFraSanity = async (): Promise<ITekstinnhold> => {
   const tekst = await sanityKlient.fetch<SanityDokument[]>(
-    '*[ytelse == "BARNETRYGD"]',
+    '*[ytelse == "BARNxRYGD"]',
   );
```

Comment:
Liten feil her?

Reply:
Er sånn vi sjekket for om feilsiden funket bare, men rettet opp i mergen nå her ☺️ :[e7d92be](https://github.com/bekk/nav-familie-endringsmelding/pull/45/commits/e7d92bed129446d37b55fcd24c3e89e263a884ca)